src/core_modules/capture-core/components/WidgetEventSchedule/WidgetEventSchedule.container.js

```diff
@@ -45,13 +46,15 @@ export const WidgetEventSchedule = ({
     });
     const { currentUser, noteId } = useNoteDetails();
     const [scheduleDate, setScheduleDate] = useState('');
+    const dateFormat = systemSettingsStore.get().dateFormat;
+    const formatDateForServer = date => moment(date, dateFormat).format('YYYY-MM-DD');
+    const formattedScheduleDate = formatDateForServer(scheduleDate);
```

Comment:
Hi @henrikmv 
Can you use existing converters instead, in the same way as is currently done elsewhere, i.e. [getConvertedRelatedStageEvent](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/WidgetEnrollmentEventNew/Validated/getConvertedRelatedStageEvent/getConvertedRelatedStageEvent.js#L54)?

```
const convertFn = pipe(convertFormToClient, convertClientToServer);
const formattedScheduleDate = convertFn(scheduleDate, dataElementTypes.DATE);
```

Thanks!